Doxyfile

```diff
@@ -32,19 +32,19 @@ DOXYFILE_ENCODING      = UTF-8
 # title of most generated pages and in a few other places.
 # The default value is: My Project.
 
-PROJECT_NAME           = "JWT-C++"
+PROJECT_NAME           = JWT-CPP
 
 # The PROJECT_NUMBER tag can be used to enter a project or revision number. This
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = 0.5.0
+PROJECT_NUMBER         = v0.5.0
```

Comment:
I think we are already at v0.5.1 with 0.6 in prerelease, so we need to update this.

Reply:
```suggestion
PROJECT_NUMBER         = v0.5.1
```